recipe/meta.yaml

```diff
@@ -39,11 +39,12 @@ requirements:
     - zlib
   run:
     - python
+    # zlib needed to be compatible with cxx-compiler
+    - zlib
```

Comment:
@chrisburr Can you confirm that I'm not committing a conda packaging anti-pattern here?

Reply:
You should be depending on `libzlib` in `host` if you're linking against it. You then don't need anything in `run` as the dependency will be added automatically by `libzlib`'s `run_exports`.

Reply:
You should also re-render after changing this.

Reply:
Attempt two... 😄

This is a long-standing debate around how to handle development dependencies. Pragmatically, I think this is the best solution for this case.

Reply:
Thanks very much @chrisburr for the fast review!

If you have references on how to be better informed on this for the future those are quite welcome. :)